gtsam/base/Matrix.h

```diff
@@ -76,6 +76,18 @@ GTSAM_MAKE_MATRIX_DEFS(9);
 typedef Eigen::Block<Matrix> SubMatrix;
 typedef Eigen::Block<const Matrix> ConstSubMatrix;
 
+// Matrix formatting arguments when printing. Akin to Matlab style.
+const Eigen::IOFormat matlab(
+  Eigen::StreamPrecision, // precision
+  Eigen::DontAlignCols, // flags set such that rowSpacers are not added
+  ", ", // coeffSeparator
+  ";\n", // rowSeparator
+  "\t",  // rowPrefix
+  "", // rowSuffix
+  "[\n", // matPrefix
+  "\n]" // matSuffix
+);
+
```

Comment:
Hmmm... I'm not comfortable with this symbol defined here in the header. It was safer as it was before, as a static var inside a smaller scope :-) 
My understanding (and experience) is that it *might* lead to link errors in some compilers/build types. 

Why not defining it as a static variable in the corresponding .cpp file, and make this a getter?

```
// In the .h

// Matrix formatting arguments when printing. Akin to Matlab style.
const Eigen::IOFormat &matlabFormat(); 
```

```
// in the .cpp

static const Eigen::IOFormat matlab(
      Eigen::StreamPrecision, // precision
      0, // flags
      ", ", // coeffSeparator
      ";\n", // rowSeparator
      "\t",  // rowPrefix
      "", // rowSuffix
      "[\n", // matPrefix
      "\n]" // matSuffix
      );

const Eigen::IOFormat &gtsam::matlabFormat() { return matlab; }
```

Reply:
That's an excellent point I had not considered, and the proposed solution is elegant! I'll get on it.